src/style.css

```diff
@@ -0,0 +1,118 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin-top: 0;
+}
+
+.form {
+  width: 100%;
+}
+
+.search {
+  position: relative;
+  display: inline-block;
+  width: 100%;
+  height: 70px;
+  box-sizing: border-box;
+  margin: 20px 0 0;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.image {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 25px;
+  top: 24px;
+}
+
+.input {
+  position: absolute;
+  left: 61.2px;
+  top: 22.4px;
+  right: 18px;
+  font-weight: 400;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  border: hidden;
+}
+
+.search--small {
+  position: relative;
+  width: 100%;
+  height: 42px;
+  margin: 16px 0 0;
+}
+
+.image--small {
+  position: absolute;
+  left: 12px;
+  top: 14px;
+  width: 11px;
+  height: 11px;
+}
+
+.input--small {
+  position: absolute;
+  left: 32px;
+  top: 8px;
+  right: 8px;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+  border: hidden;
+}
+
+.input::placeholder {
+  font-family: Avenir, Helvetica, sans-serif;
+  color: #3d4e61;
+  font-size: 16px;
+  font-weight: 500;
+}
+
+.input--small::placeholder {
+  font-family: Avenir, Helvetica, sans-serif;
+  color: #3d4e61;
+  font-size: 14px;
+  font-weight: 500;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 3px;
+}
+
+.input:hover {
+  font-weight: bold;
```

Comment:
font-weight should change on focus, not hover
<img width="263" alt="image" src="https://user-images.githubusercontent.com/66749919/200109242-82b7ba5e-972e-4a2d-b550-da8a1fa2c5d3.png">